std/array.d

```diff
@@ -796,22 +796,49 @@ private void copyBackwards(T)(T[] src, T[] dest)
 
     assert(src.length == dest.length);
 
-    if (!__ctfe || hasElaborateCopyConstructor!T)
+    static if (!hasElaborateAssign!T)
     {
-        /* insertInPlace relies on dest being uninitialized, so no postblits allowed,
-         * as this is a MOVE that overwrites the destination, not a COPY.
-         * BUG: insertInPlace will not work with ctfe and postblits
-         */
-        memmove(dest.ptr, src.ptr, src.length * T.sizeof);
+        if (!__ctfe)
```

Comment:
`if (!__ctfe)` is special. It's a static if while not being an actual static if. I believe `__ctfe` cannot even be accessed via a `static if`